gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/DefaultPubSubRpc.java

```diff
@@ -69,12 +69,12 @@ public DefaultPubSubRpc(PubSubOptions options) throws IOException {
     try {
       // Provide (and use a common thread-pool).
       // This depends on https://github.com/googleapis/gax-java/issues/73
-      PublisherSettings.Builder pbuilder = PublisherSettings.defaultInstance().toBuilder();
+      PublisherSettings.Builder pbuilder = PublisherSettings.defaultBuilder();
```

Comment:
Could also `pbuilder.provideChannelWith(credentials)` be used instead of
```
pbuilder.provideChannelWith(ConnectionSettings.newBuilder()
    .provideCredentialsWith(credentials).build());
```